Documentation/installation/kind.rst

```diff
@@ -48,36 +48,30 @@ Then, install Cilium release via Helm:
 
 .. note::
 
-   To enable Cilium's kube-proxy replacement (:ref:`kubeproxy-free`), kind nodes
-   need to have their own cgroup namespaces, and are different from the cgroup namespace
+   To enable Cilium's Socket LB (:ref:`kubeproxy-free`), cgroup v2 needs to be
+   enabled, and Kind nodes need to run in separate `cgroup namespaces
```

Comment:
>cgroup v2 needs to be
   enabled, and Kind nodes need to run in separate `cgroup namespaces

nit: Per the docker container runtimes documentation, we can keep the terminology consistent and call this **cgroups** v2 (this suggests cgroup v2 controllers).

Reply:
Actually, K8s is using "cgroup v2" (https://kubernetes.io/docs/concepts/architecture/cgroups/). Also, in other documentation pages of Cilium we use "cgroup v2". I am sticking to "cgroup v2".